src/applications/financial-status-report/utils/streamlinedDepends.js

```diff
@@ -0,0 +1,114 @@
+import { DEBT_TYPES } from '../constants';
+
+const VHA_LIMIT = 5000;
+
+/**
+ * @param {object} formData - all formData
+ * @returns true if the following conditions are met:
+ * - Streamlined wiaver feature flag is true
+ * - Only copays have been selected
+ * - Selected copay balances are below $5,000
+ */
+export const isEligibleForStreamlined = formData => {
+  if (formData?.selectedDebtsAndCopays?.length === 0) return false;
+
+  return (
+    formData['view:streamlinedWaiver'] &&
+    formData?.selectedDebtsAndCopays?.every(
+      debt => debt.debtType === DEBT_TYPES.COPAY,
+    ) &&
+    formData?.selectedDebtsAndCopays?.reduce(
+      (acc, copay) => acc + copay?.pHAmtDue,
+      0,
+    ) < VHA_LIMIT
+  );
+};
+
+/**
+ * @param {object} formData - all formData
+ * @returns true if the following conditions are met:
+ * - isElidgeibleForStreamlined is true
```

Comment:
typo: "isElidgeibleForStreamlined." It should be corrected to "isEligibleForStreamlined."